linux/keyman-config/tests/test_canonical_language_code_utils.py

```diff
@@ -21,8 +21,8 @@ def test_FindBestTag_ValidCases(self):
             { 'expected': 'sqt-Arab-YE', 'tags': ['sqt', 'sqt-YE', 'sqt-Arab'] },
             { 'expected': 'sqt-Latn-YE', 'tags': ['sqt-Latn', 'sqt-Latn-YE'] },
 
-            { 'expected': 'sa-Latn', 'tags': ['sa-Latn'] },
-            { 'expected': 'hi-Latn', 'tags': ['hi-Latn'] },
+            { 'expected': 'sa-Latn', 'tags': ['sa-Latn'] }, # Should sa-Latn go away?
+            { 'expected': 'hi-Latn-IN', 'tags': ['hi-Latn'] },
```

Comment:
Should this be 
`{ 'expected': 'hi-Latn-IN', 'tags': ['hi-Latn', 'hi-Latn-IN'] }`

Reply:
The other tests cases seem to combine the `"tag"` and `"tags"` fields. In this case, there's only
```
  "tag": "hi-Latn",
```

so maybe that's all we test?

Reply:
I don't understand why you've added `-IN` here?

Reply:
langtags.json has a new entry for `hi-Latn-IN`. See
https://github.com/keymanapp/keyman/pull/6594#discussion_r868628392

Reply:
I need to read up on the lang tags and the code this was just a comment observing the tests vs the json file of other tags..  I think it should test that it matches both. I see what you say about the tag and tags fileds, but if you look at `sqt-Latn-YE` it only has `sqt-Latn` in the tag, but tests for ``sqt-Latn-YE` also.

Reply:
I looked a bit deeper. `hi-Latn` is a new block in this langtags.json (it is not present in the beta branch version at all). So we need to understand why this test was present -- perhaps we should test with a different script?

Reply:
afaict, the unit test was introduced in #3545 to address this reported keyboard issue involving `sa-Latn` and `hi-Latn`

https://github.com/keymanapp/keyboards/pull/753#issuecomment-503832382